queries/boundaries.jinja2

```diff
@@ -84,6 +85,7 @@ SELECT
   'yes' AS maritime_boundary,
   NULL AS admin_level,
   NULL AS boundary_type,
+  NULL AS kind,
```

Comment:
I'm not familiar with the data, does it make sense to fold the `boundary_type` into the `kind` here?